src/Components/Admin/navbar.js

```diff
@@ -47,13 +46,15 @@ export default function Navbar({ home }) {
 
     async function handleLogout() {
         handleClose();
-        history.push("/login");
         auth.signOut();
         await httpGet(
             process.env.REACT_APP_ADMIN_BASE_ENDPOINT + "admin_logout"
         );
+        history.push("/login");
```

Comment:
Why is this reorder needed?

Reply:
Since the routing is now dependent on isAdmin, which gets recomputed on auth state changes, it is needed to be after the auth.signOut and the resetting of the cookies which the admin_logout does.